api/src/test/java/org/openmrs/module/ugandaemr/RolePrivilegeImportTest.java

```diff
@@ -15,9 +16,16 @@ public class RolePrivilegeImportTest extends BaseModuleContextSensitiveTest {
 
     protected static final String ROLE_PRIVILLEGE_DATASET_XML = "metadata/Role_Privilege.xml";
 
+    UserService userService=null;
```

Comment:
@slubwama you do not need the above line if you do the changes in my patch.

Reply:
@dkayiwa true and it has made the tests run well. though I am now getting a run time error  found in this stack trace https://pastebin.com/5xRPRyJr which causes the server to fail to start

Reply:
This is the cause of the error `ClassNotFoundException: org.springframework.web.servlet.mvc.annotation.DefaultAnnotationHandlerMapping`. Just remove it from the spring configuration files of your modules.

Reply:
@dkayiwa thanks. It has started Now going to test functionality in all modules